### PR TITLE
Dev/207 Tags API integrate

### PR DESCRIPTION
**Pull request by Chaoyuuu**
Solved #207 

image

### CODE REVIEW

File: src/app/problem/editor/right/Tags.js [hunk 1]
```diff
@@ -1,26 +1,46 @@
...
     const [isEditing, setIsEditing] = useState(false);
     const [textItemsBackUp, setTextItemsBackUp] = useState(undefined);
     const {textItems, setTextItems, addTextItem, removeTextItem} = useTextItems();
 
+    useEffect(() => {
+        if (!problem) {
+            problemService.getProblemById(problemId)
+                .then(p => {
+                    setProblem(p);
+                    setTextItems(p.tags.map(tag => new TextItem(tag)))
```
Comment (Wally5077) [inline src/app/problem/editor/right/Tags.js:27]: ```;```

**Review by Wally5077 — CHANGES_REQUESTED** [review]
Review done

**Comment by Chaoyuuu** [comment]
Hi, I have changed the CSS. Please review again. Thanks! 😄

File: src/app/problem/ProblemList.js [hunk 2]
```diff
@@ -48,27 +47,21 @@ const ProblemList = () => {
                                   Button={() =>
                                       <CreateButton onClick={() => setShowCreateProblemModal(true)}/>}
                                   tableHeaders={[
-                                      <TableCell>#</TableCell>,
-                                      <TableCell>Problem Title</TableCell>,
-                                      <TableCell>Tags</TableCell>
+                                      <div>#</div>,
+                                      <div>Problem Title</div>,
+                                      <div>Tags</div>
```
Comment (Wally5077) [inline src/app/problem/ProblemList.js:52]: NIT: Why did you use ```div``` here?

**Review by Wally5077 — APPROVED** [review]
LGTM

**Review by Johnny850807 — APPROVED** [review]
LGTM

**Pull request merged**